chore: added LGTM badges to readme (#1380)

XDRAGON2002 opened the pull request:
Closes #1380

terriko reviewed (changes requested):
Looks like you need to add 'lgtm' to the allowed spelling list.

lgtm offers two possible badges.  I think the "code quality" one would probably be more meaningful to people who aren't us, so maybe we could use that one instead of the lgtm alerts number one? 

Here's the code for the code quality one:
[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/intel/cve-bin-tool.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/intel/cve-bin-tool/context:python)

```
[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/intel/cve-bin-tool.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/intel/cve-bin-tool/context:python)
```

XDRAGON2002 commented:
Added "lgtm" to list of accepted words.
I had actually added both, the alerts badge as well as the language grade badge, so as requested, removed the alerts badge.

terriko reviewed (approved):
Thanks!
